_episodes/08-environment-variables.md

```diff
@@ -12,15 +12,17 @@ objectives:
 keypoints:
 - "Shell variables are by default treated as strings"
 - "The `PATH` variable defines the shell's search path"
-- "Variables are assigned using \"`=`\" and recalled using the variable's name prefixed by \"`$`\""
+- "Variables are assigned using \"`=`\" and recalled using the variable's name 
+prefixed by \"`$`\""
```

Comment:
```suggestion
  prefixed by \"`$`\""
```

Reply:
I've found that a leading space like this isn't necessary. Jekyll renders a space when there is a line break in the text.